lib/stormpath-sdk/resource/application.rb

```diff
@@ -117,6 +118,10 @@ def get_provider_account request
   def oauth_authenticate(request)
     Stormpath::Oauth::Authenticator.new(data_store).authenticate(href, request) 
   end
+  
+  def create_login_attempt(options = {})
```

Comment:
I didn’t understood your intentions at first. I made the changes, the api is as You described it above.